cost/cheaper_regions/cheaper_regions.pt

```diff
@@ -201,10 +227,10 @@ policy "azure_cheaper_regions" do
     summary_template "{{ rs_project_name }} (Account ID: {{ rs_project_id }}): {{ len data }} rows with cheaper data"
     detail_template <<-EOS
 # Cheaper Regions
-| Billing Center Name | Billing Center ID | Region In Use | Suggested Region | Service |
-| ------------------- | ----------------- | ------------- | ---------------- | ------- |
+| Billing Center Name | Billing Center ID | Region In Use | Suggested Region | Service | Vendor |
+| ------------------- | ----------------- | ------------- | ---------------- | ------- | ------ |
 {{ range data -}}
-| {{ .billing_center_name }} | {{ .billing_center_id }} | {{ .region }} | {{ .new_region }} | {{.service}} |
+| {{ .billing_center_name }} | {{ .billing_center_id }} | {{ .region }} | {{ .new_region }} | {{.service}} | {{.vendor}} |
 {{ end -}}
 EOS
```

Comment:
Please add the following footer to the incident:
`___`
`###### Policy Applied in Account: {{ rs_project_name }} (Account ID: {{ rs_project_id }}) within Org: {{ rs_org_name }} (Org ID: {{ rs_org_id }})`